src/pages/home/ReportScreen.js

```diff
@@ -117,6 +127,9 @@ class ReportScreen extends React.Component {
 
     componentWillUnmount() {
         clearTimeout(this.loadingTimerId);
+        if (window.visualViewport) {
+            window.visualViewport.removeEventListener('resize', this.viewportOffsetTop);
```

Comment:
@neil-marcellini nice work on this PR and feature. I came across this use of `window` in a cross-platform component and wanted to call out [this section of the README](https://github.com/expensify/app#platform-specific-file-extensions:~:text=Note%20that%20index.js%20should%20be%20the%20default%20and%20only%20platform%2Dspecific%20implementations%20should%20be%20done%20in%20their%20respective%20files).

We really shouldn't be seeing `window` unless we are in an `index.js` file and there's an `index.native.js` that is doing a no-op.

Reply:
Ok, got it. Is it worth making an `index.native.js` version of this component to fix this? Or could I make a `withViewportOffsetTop` HOC that would have a web and native version?

Reply:
Both could work. Though, this component is pretty important and having two might get confusing. I would create a small lib that returns something like `VisualViewport` we can call `addEventListener()` on and for mobile it would just return an empty function.

Reply:
Could also return a cleanup method something like 

**index.js**
```js
function addResizeListener(callback) {
    if (!window.visualViewport) {
        return () => {};
    }

    window.visualViewport.addEventListener('resize', callback);
    return () => window.visualViewport.removeEventListener('resize', callback);
}
```

**index.native.js**
```js
function addResizeListener(callback) {
    return () => {};
}
```

I feel like I've used this pattern before elsewhere to get around the cross-platform guidance.